test/order_test.rb

```diff
@@ -7,7 +7,7 @@
 
 Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
 
-describe "Order Wave 1" do
+xdescribe "Order Wave 1" do
```

Comment:
You shouldn't ignore tests once they pass, since part of their value is in catching if you introduce a new bug.